### PR TITLE
chore(website): add tooltip vs popover video, some other cleanup

### PR DESCRIPTION
**Pull request by TheSisb**
- [x] Add tooltip vs popover video to both pages
- [x] Clean up and enhance examples on tooltip and popover pages
- [x] Add mention to breadcrumb docs to avoid trailing slashes

### CODE REVIEW

File: packages/paste-website/src/pages/components/popover/index.mdx [hunk 1]
```diff
@@ -75,6 +78,19 @@ export const pageQuery = graphql`
 The Popover component is a non-modal dialog that is commonly used for displaying additional rich content
 above your UI.
 
+### Popover compared to Tooltip
+
+<div style="position: relative; padding-bottom: 56.25%; height: 0;">
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/components/popover/index.mdx:83]: Are you trying to maintain aspect ratio but not using your aspect ratio component?

File: packages/paste-website/src/pages/components/tooltip/index.mdx [hunk 2]
```diff
@@ -133,10 +146,10 @@ A tooltip **must** be placed on a **focusable** element, such as an anchor or bu
   }}
 >
   {`<Box display="flex" alignItems="center">
-  <Text as="span">Tooltip should appear from the icon.</Text>
-  <Tooltip text="Welcome to Paste!">
+  <Text as="span">Social Security Number</Text>
+  <Tooltip text="Your nine-digit Social Security number is your first and continuous connection with Social Security.">
     <Anchor href="https://paste.twilio.design">
-      <InformationIcon decorative={false} title="Open Tooltip" display="block" />
+      <InformationIcon decorative={false} title="More information icon" display="block" />
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/components/tooltip/index.mdx:152]: The title here is acting as the text content of the anchor. We should be trying to describe what the anchor is doing, not what the icon is in this case

File: packages/paste-website/src/pages/components/popover/index.mdx (hunk 1 above)
Comment (TheSisb) [reply packages/paste-website/src/pages/components/popover/index.mdx:83]: I guess it is, this was just a copy/pasted embed code from Loom.

File: packages/paste-website/src/component-examples/TooltipExamples.ts [hunk 3]
```diff
@@ -9,10 +9,10 @@ const StateHookExample = () => {
       >
         <Button variant="primary">Open tooltip</Button>
       </Tooltip>
-      <Button variant="primary" onClick={() => tooltip.show()}>
+      <Button variant="secondary" onClick={() => tooltip.show()}>
         Open tooltip on click
       </Button>
-      <Button variant="primary" onClick={() => tooltip.hide()}>
+      <Button variant="secondary" onClick={() => tooltip.hide()}>
```
Comment (richbachman) [inline packages/paste-website/src/component-examples/TooltipExamples.ts:15]: Good catch here!

File: packages/paste-website/src/pages/components/popover/index.mdx (hunk 1 above)
Comment (richbachman) [reply packages/paste-website/src/pages/components/popover/index.mdx:83]: It'd be worth seeing if Aspect-Ratio works here. I don't think we've used it too often in the wild.

File: packages/paste-website/src/pages/components/popover/index.mdx [hunk 4]
```diff
@@ -75,6 +79,18 @@ export const pageQuery = graphql`
 The Popover component is a non-modal dialog that is commonly used for displaying additional rich content
 above your UI.
 
+### Popover compared to Tooltip
+
+<AspectRatio ratio="16:9">
```
Comment (richbachman) [inline packages/paste-website/src/pages/components/popover/index.mdx:84]: 🔥

**Review by richbachman — APPROVED** [review]

**Pull request merged**